labs/2-perpetual-exchange/2-trading.md

```diff
@@ -0,0 +1,97 @@
+
+
+## Take-Profit / Stop-Loss Orders
+
+You can set take-profit and stop-loss trigger orders by selecting the `Add TP` or `Add SL` button in the position row.
+
+Once you create a trigger order, it will be displayed in your position's row under the `Take Profit` and `Stop Loss` columns. You have the flexibility to edit the order and adjust the trigger price as needed.
+
+In the event of manually closing a position, the associated trigger orders will be automatically canceled and closed, providing a seamless experience when opening new future positions.
+
+*Note: While `TP/SL` orders are set, keepers will monitor the mark price, when reaching the specified price level, TP/SL will close the whole position.*
+
+## Liquidations
+
+In the scenario of opening a long position in SOL, where the position size surpasses the collateral value, there exists a critical point known as the Liquidation Price. This price is calculated based on the threshold where the loss amount, collateral value, and borrow fee collectively dip below 0.5% of your position's size. If the token's price surpasses this point, the system will automatically close the position.
+
+It's crucial to note that the liquidation price is subject to change over time, particularly with leverage exceeding 10x and extended position durations. Regularly monitoring your liquidation price is essential. To mitigate the risk of liquidation, collateral adjustments and leverage fine-tuning can be performed through the `Edit` button in the position row, offering an avenue to add collateral and enhance the liquidation price.
+
+Upon liquidation, any remaining collateral, net of losses and fees, will be returned to your account.
+
+## Pricing
+
+On Jupiter Perpetual Exchange, **trades incur no price impact**, allowing for the execution of large trades precisely at the mark price. 
+
+Mark prices are conveniently presented in the same row as the market name *(e.g., SOL-PERP)*. Notably, long positions and short positions are opened and closed at the same price as `Mark Price` when initiated market order. 
+
+## Fees
+
+Opening or closing a position on Jupiter Perpetual Exchange incurs a fee equivalent to 0.1% of the position size.
+
+For long positions, the collateral is the token being longed *(e.g., SOL for SOL longs, BTC for BTC longs)*, while short positions are collateralized with supported stablecoins like USDC or USDT.
+
+Jupiter Perpetuals Trading seamlessly integrates Jupiter Swap. Consequently, when a swap is required during the opening or closing of a position, regular DEX fees and slippages are applicable. The specific fee is contingent on whether the swap enhances or diminishes the balance.
```

Comment:
> contingent on whether the swap enhances or diminishes the balance

don't quite understand this part

Reply:
This part is a bit confusing yes. Let me remove it.